src/SDK/Common/Retry/ExponentialWithJitterRetryPolicy.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenTelemetry\SDK\Common\Retry;
+
+use InvalidArgumentException;
+use OpenTelemetry\SDK\Common\Time\BlockingScheduler;
+use OpenTelemetry\SDK\Common\Time\SchedulerInterface;
+use OpenTelemetry\SDK\Metrics\Exceptions\RetryableExportException;
+use OpenTelemetry\SDK\Trace\SpanExporterInterface;
+
+final class ExponentialWithJitterRetryPolicy implements RetryPolicyInterface
+{
+    private $maxAttempts;
+    private $initialBackoff;
+    private $maxBackoff;
+    private $backoffMultiplier;
+    private $jitter;
+    public $retryableStatusCodes;
+    public $scheduler;
+
+    public function __construct($arguments=[])
+    {
+        extract($arguments, EXTR_IF_EXISTS);
```

Comment:
this extracts the arguments into local variables, however the code just gets them from the array anyway?

Reply:
The build will fail for php 7.4 for now. We can refactor the same after we drop the support for php 7.4. I hope thats fine

Reply:
I assume we will reach beta stability before EOL of 7.4, refactoring then would be a BC-break.
Adding the parameters shouldn't break 7.4, it just requires users to provide default values explicitly if they want to override later arguments.

Reply:
Done